src/wp-admin/css/customize-controls.css

```diff
@@ -362,11 +362,16 @@ body.trashing #publish-settings {
 .date-time-fields .date-input.day,
 .date-time-fields .date-input.hour,
 .date-time-fields .date-input.minute {
-	width: 46px;
+	width: 52px;
```

Comment:
Instead of adding another 5 or 6 pixels to each input's width, you could add the `tiny-text` class on each of [these number fields](https://core.trac.wordpress.org/browser/trunk/src/wp-includes/customize/class-wp-customize-date-time-control.php?rev=55276&marks=157,167,185,193#L151). The `forms.css` stylesheet already [removes the right padding](https://core.trac.wordpress.org/browser/trunk/src/wp-admin/css/forms.css?rev=59046&marks=482,491#L480) to make room for the increment/decrement buttons. 

(`small-text` might seem more appropriate for the Year field, but it still has [padding on small screens](https://core.trac.wordpress.org/browser/trunk/src/wp-admin/css/forms.css?rev=59046&marks=1674-1685#L1684) and the Customizer's width works with `tiny-text`)

Reply:
I like the idea to add the class and apply CSS their.